README.md

```diff
@@ -269,24 +269,17 @@ Have you used the theme on your website? Send a PR to add it to the list for ins
 	<tbody>
 		<tr>
             <td align="center">
-                <a href="https://github.com/zetxek">
```

Comment:
I'm not sure the README has to be updated - can you check if it can be excluded from the PR?

This section is automatically generated by a GitHub action

Reply:
i just ignored this and let the action run again after merging :) 
https://github.com/zetxek/adritian-free-hugo-theme/actions/runs/14036430147